File: code/JL_OTA_SDK/otasdk_java/src/main/java/com/jieli/otasdk_java/base/BaseActivity.java
package com.jieli.otasdk_java.base;

import android.os.Bundle;

import androidx.annotation.Nullable;

import com.blankj.utilcode.util.BarUtils;
import com.jieli.component.base.Jl_BaseActivity;

/**
 * @ClassName: BaseActivity
 * @Description: java类作用描述
 * @Author: ZhangHuanMing
 * @CreateDate: 2021/12/23 9:32
 */
public class BaseActivity extends Jl_BaseActivity {
    @Override
    public void onCreate(@Nullable Bundle bundle) {
        super.onCreate(bundle);
        BarUtils.setStatusBarLightMode(this, true);
    }
}

File: code/JL_OTA_SDK/otasdk/src/main/java/com/jieli/otasdk/base/BaseActivity.kt
package com.jieli.otasdk.base

import android.os.Bundle
import com.blankj.utilcode.util.BarUtils
import com.jieli.component.base.Jl_BaseActivity

/**
 *  create Data:2019-07-24
 *  create by:chensenhua
 *
 **/
open class BaseActivity : Jl_BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        BarUtils.setStatusBarLightMode(this,true)

    }


}